pkgs/top-level/all-packages.nix

```diff
@@ -6926,7 +6926,7 @@ in
 
   ssdeep = callPackage ../tools/security/ssdeep { };
 
-  ssh-ident = callPackage ../tools/networking/ssh-ident { };
+  ssh-ident = python3.pkgs.callPackage ../tools/networking/ssh-ident { };
```

Comment:
Don't use `python3.pkgs.callPackage`. It is done elsewhere but should not be done because it does not compose.

Reply:
so what is the correct way to migrate to python3 ?

Reply:
just `callPackage`